post-processor/vagrant-cloud/step_create_provider.go

```diff
@@ -46,6 +46,10 @@ func (s *stepCreateProvider) Run(ctx context.Context, state multistep.StateBag)
 	if err != nil || (resp.StatusCode != 200) {
 		cloudErrors := &VagrantCloudErrors{}
 		err = decodeBody(resp, cloudErrors)
+		if err != nil {
+			state.Put("error", fmt.Errorf("error decoding error response: %s", err))
+			return multistep.ActionHalt
```

Comment:
This is shadowing the original error that we emit line 53; which should be more important, I believe; here, just logging the error should be sufficient.

ditto throughout.

Reply:
Done.

Reply:
Awesome; thanks ! 👍